example/lib/pages/mini_games/multi_scenario/maps/map_biome_1.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:bonfire/bonfire.dart';
+import 'package:example/pages/mini_games/multi_scenario/multi_scenario.dart';
+import 'package:example/pages/mini_games/multi_scenario/player/game_player.dart';
+import 'package:example/pages/mini_games/multi_scenario/player/sprite_sheet_hero.dart';
+import 'package:example/pages/mini_games/multi_scenario/utils/constants/game_consts.dart';
+import 'package:example/pages/mini_games/multi_scenario/utils/enums/map_id_enum.dart';
+import 'package:example/pages/mini_games/multi_scenario/utils/enums/show_in_enum.dart';
+import 'package:example/pages/mini_games/multi_scenario/utils/sensors/exit_map_sensor.dart';
+import 'package:flutter/material.dart';
+
+class MapBiome1 extends StatelessWidget {
+  final ShowInEnum showInEnum;
+
+  const MapBiome1({Key? key, this.showInEnum = ShowInEnum.left})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+        future: SpriteSheetHero.load(),
+        builder: (context, snapshot) {
+          if (!snapshot.hasData) {
+            return const SizedBox.shrink();
+          }
+          return BonfireWidget(
+            joystick: Joystick(
+              keyboardConfig: KeyboardConfig(),
+              directional: JoystickDirectional(),
+            ),
+            player: GamePlayer(
+              _getInitPosition(),
+              SpriteSheetHero.hero1,
+              initDirection: showInEnum.direction,
+            ),
+            map: WorldMapByTiled(
+              MultiScenarioAssets.mapBiome1,
+              forceTileSize: Vector2.all(defaultTileSize),
+              objectsBuilder: {
+                'sensorLeft': (p) => ExitMapSensor(
```

Comment:
The sensors are not working (map doesn't change and player stops moving), at least not on the web in release mode.